### PR TITLE
implement error object on multiple exercises

### PR DESCRIPTION
**Pull request by rpottsoh**
per #1311
phone-number
say
variable-length-quantity
queen-attack
forth

### CODE REVIEW

File: exercises/say/canonical-data.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "exercise": "say",
-  "version": "1.1.0",
+  "version": "1.2.0",
   "comments": [
     "Here -1 is used as expected value to indicate that the",
```
Comment (sdublish) [inline exercises/say/canonical-data.json]: Would it make sense to update the comments to reflect the new error object?

**Comment by Insti** [comment]
Multiple changes in 1 PR makes reviewing harder.
Make sure you check all the comments at the start of the files.

File: exercises/say/canonical-data.json (hunk 1 above)
Comment (rpottsoh) [reply exercises/say/canonical-data.json]: Yes it would.  :smile:   Thanks for pointing it out.

**Comment by rpottsoh** [comment]
> Multiple changes in 1 PR makes reviewing harder.

Sorry about that.  I hadn't considered that.

**Pull request merged**